### PR TITLE
Expose `Data.Generics.Labels` module

### PR DESCRIPTION
**Pull request by jgertm**
I needed this to throw out my dependency on `generic-lens-labels`.

### CODE REVIEW

**Comment by kcsongor** [comment]
Yes, that module should probably be removed from here, but it looks like `generic-lens-labels` is quite outdated these days

**Comment by jgertm** [comment]
@mpickering: I agree that it's a bad orphan instance, but shouldn't users of this package have the choice to opt into it by importing the module?

@kcsongor: `generic-lens-labels` is indeed very outdated, which is what motivated this PR in the first place. To me it makes sense to include the instance in this package so that it remains up-to-date, make it opt in and add a big disclaimer at the top.

In summary: [what they said](https://github.com/kcsongor/generic-lens/issues/16#issuecomment-344445288)

**Comment by kcsongor** [comment]
Hi @jgertm, thanks for submitting this patch, the module is now exported! (since `1.1.0.0`)
